helm/cluster-api-core/Chart.yaml

```diff
@@ -4,3 +4,5 @@ description: "Helm chart for cluster api core."
 home: "https://github.com/giantswarm/cluster-api-core-app"
 version: "[[ .Version ]]"
 appVersion: 0.3.19-gs1
+annotations:
+  application.giantswarm.io/team: "firecracker"
```

Comment:
Wouldn't this change make alerts in Azure installations go to firecracker as well? is that expected?

Reply:
> Wouldn't this change make alerts in Azure installations go to firecracker as well? is that expected?

@fiunchinho Yes, but there is also the owners annotation that is a bit fancier and lets us route by provider.

https://intranet.giantswarm.io/docs/dev-and-releng/app-developer-processes/routing_app_alerts/#owners-annotation

Should I change it to that?

Reply:
I think so, but maybe I'm missing something. What other people think?

Reply:
Jose, I'd like to get this merged and currently the app is only pushed to the AWS app collection.

We can easily switch to the owners annotation when it's also deployed to Azure MCs.